direct/src/gui/OnscreenText.py

```diff
@@ -300,10 +301,12 @@ def setTextX(self, x):
         self.setTextPos(x, self.__pos[1])
 
     def setX(self, x):
-        """
+         """
```

Comment:
I think this is the erroneous space causing the checks error

Reply:
fixed